data/hai/unfettered hunger.txt

```diff
@@ -0,0 +1,425 @@
+			`	"Human," he says again when he reaches you, "you should not promenade yourself in our star market without worthy guidance. This warrior can tell you what trader tongues are saying because I know your backward speech. Fast fingers not grab human things while I watch," he adds, emphasizing the point by banging the staff on the floor and making it ring. You notice the flow of traffic begins to part around you as other traders and travelers give him room.`
+			`	It takes you a moment to parse what he is saying as his accent is difficult to understand. You would guess he learned your language in class but has never spoken with humans. You think he is offering to be your translator and bodyguard.`
+			choice
+				`(Hire him.)`
+					goto hire
+				`(Politely decline.)`
+					goto polite
+				`(Laugh at him and walk away.)`
+					goto laugh
+			
+			label hire
+			`	The young Unfettered proves diligent in his work. He steers you away from some vendors and towards others and the prices and quality are at least competitive. Several times his staff has flashed out and blocked people who might have bumped into you. While some of the vendors you traded with were amused by his translations, most worked through him rather than speaking to you directly. He even kept watch while you were eating, though he did seem to be distracted by the food.`
+			`	At the end of the day, he follows you to the <ship> and you pay him his fee. He surprises you though, by following you on board. "I will watch over you until you make it back to the Fettered worlds. You should be safe enough on <destination>." From the prices you saw in the restaurants and shops, the amount you paid would feed him for a few days at most.`
+			choice
+				`	(Give him transport.)`
+					goto transport
+				`	(Escort him off the ship.)`
+					goto escort
+				
+			label polite
+			`	You tell him that you can take care of yourself. He nods to acknowledge your decision but maintains eye contact. He then turns and walks away.`
+			`	You see him around the spaceport a few times: once working as a porter, another time loading cargo. Both seem like odd jobs for someone with his apparent skills and neither will pay very well. As you are heading back to your ship you see him at one of the small food stalls that sells really cheap noodle cups and you wonder if that's the only time he's eaten today.`
+				decline
+
+			label laugh
+			`	You find yourself flat on your back looking up at the young Unfettered. He's holding your sidearm and is pointing it at you with the safety off. You later decide that he must have swept your legs with the staff and grabbed your weapon, but it happened so swiftly you never saw it.`
+			`	"I was wrong," he says as he secures your weapon inside his jacket. "I thought you warrior. Have respect. You just monkey after all."`
```

Comment:
```suggestion
			`	"I was wrong," he says as he secures your weapon inside his jacket. "I thought you were a warrior. Have respect. You are just a monkey after all."`
```
To make it read better. Unless it was purposely written this way because of his accent?

Reply:
kinda on purpose yeah but I just tweaked it caus it was a bit too bad haha